### PR TITLE
Move ostree-boot-image to launch as a pod in openshift

### PR DESCRIPTION
**Pull request by p3ck**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -111,6 +111,12 @@ podTemplate(name: podName,
                         ttyEnabled: true,
                         command: 'cat',
                         privileged: true,
+                        workingDir: '/workDir'),
+                containerTemplate(name: 'ostree-boot-image',
+                        image: DOCKER_REPO_URL + '/' + OPENSHIFT_NAMESPACE + '/ostree-boot-image:' + SLAVE_TAG,
```
Comment (scoheb) [inline Jenkinsfile]: should have tag for your image...OSTREE_BOOT_IMAGE_TAG

File: Jenkinsfile [hunk 2]
```diff
@@ -111,6 +111,12 @@ podTemplate(name: podName,
...
+                        ttyEnabled: true,
+                        command: '/usr/sbin/init',
```
Comment (scoheb) [inline Jenkinsfile]: should be 'cat' for command

File: config/s2i/ostree-boot-image/ostree-boot-image-buildconfig-template.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - type: ConfigChange
+parameters:
+- description: Git repository with Dockerfile and slave entrypoint.
+  displayName: Repository URL
+  name: REPO_URL
+  value: https://github.com/CentOS-PaaS-SIG/ci-pipeline.git
+- description: The sub-directory inside the repository.
+  displayName: Context Directory
+  name: REPO_CONTEXTDIR
+  value: config/Dockerfiles/ostree-boot-image
```
Comment (scoheb) [inline config/s2i/ostree-boot-image/ostree-boot-image-buildconfig-template.yaml]: this does not match up with the actual path in repo...I see '_' in path. I think everything should be '-'

Comment (p3ck) [reply config/s2i/ostree-boot-image/ostree-boot-image-buildconfig-template.yaml]: where do you see _?

File: Jenkinsfile (hunk 1 above)
Comment (p3ck) [reply Jenkinsfile]: ok - fixing

File: Jenkinsfile (hunk 2 above)
Comment (p3ck) [reply Jenkinsfile]: pretty sure this has to be /usr/sbin/init or the libvirtd service won't start

**Comment by p3ck** [comment]
[test] lets try this..

**Comment by p3ck** [comment]
[test] doesn't look like the container rebuilt

**Comment by p3ck** [comment]
[test] -1 strikes again

**Comment by scoheb** [comment]
[test]

**Comment by p3ck** [comment]
[test]

**Pull request closed without merging**